ui/src/main/java/org/pentaho/di/ui/trans/steps/tableoutput/TableOutputDialog.java

```diff
@@ -1064,6 +1066,41 @@ private void getSchemaNames() {
     }
   }
 
+  private void validateSelection() {
+
+    Runnable fieldLoader = () -> {
+
+      isConnectionSupported();
+    };
+    shell.getDisplay().asyncExec( fieldLoader );
+  }
+
+  protected void isConnectionSupported() {
+
+    final String tableName = wTable.getText(), connectionName = wConnection.getText();
+
+    if ( !Utils.isEmpty( tableName ) ) {
+
+      DatabaseMeta ci = transMeta.findDatabase( connectionName );
```

Comment:
Why is this variable called `ci` when it is a `DatabaseMeta` instance?
Consider something like `meta`, `dbmeta`, `databaseMeta`, etc...